src/main/java/mate/academy/spring/model/Role.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.spring.model;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Enumerated(EnumType.STRING)
+    private RoleName roleName;
+
+    public enum RoleName {
+        ADMIN, USER
+    }
```

Comment:
```suggestion
```
better use string constants in configs
you dont really need this enum because there is can be a lot more roles, and it should be stored in db

Reply:
such a condition is in a task
![зображення](https://user-images.githubusercontent.com/85991807/194477932-b2c23241-64ea-4130-9fdb-3b430b773c2a.png)